babel.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       },
     ],
     '@babel/preset-typescript',
-    'module:metro-react-native-babel-preset',
+    'module:@react-native/babel-preset',
   ],
   plugins: [
     '@babel/plugin-proposal-class-properties',
```

Comment:
Removing these two lines fixes the issue
```
'loose' mode configuration must be the same for @babel/plugin-transform-class-properties, @babel/plugin-transform-private-methods and @babel/plugin-transform-private-property-in-object (when they are enabled).
```
but makes 2 snapshot tests fail.

Reply:
🤯